constants/fallFest2022Content.ts

```diff
@@ -0,0 +1,273 @@
+    type: 'Olivia Lanes',
+    url: 'Lecture'
+  },
+  {
+    day: 'Monday, July 18',
+    university: 'Q&A Sessions',
+    type: '-',
+    url: 'Live Q&A'
+  },
+  {
+    day: 'Tuesday, July 19',
+    university: 'Introduction to Linear Algebra, Prerequisite Mathematics, and Circuit Composition',
+    type: 'Maria Violaris',
+    url: 'Lecture'
+  },
+  {
+    day: 'Tuesday, July 19',
+    university: 'Q&A Sessions',
+    type: '-',
+    url: 'Live Q&A'
+  },
+  {
+    day: 'Wednesday, July 20',
+    university: 'Hamiltonian Time Evolution',
+    type: 'Maria Violaris',
+    url: 'Lecture'
+  },
+  {
+    day: 'Wednesday, July 20',
+    university: 'Q&A Sessions',
+    type: '-',
+    url: 'Live Q&A'
+  },
+  {
+    day: 'Thursday, July 21',
+    university: 'Simulation Problems',
+    type: 'Jeffrey Cohn',
+    url: 'Lecture'
+  },
+  {
+    day: 'Thursday, July 21',
+    university: 'Q&A Sessions',
+    type: '-',
+    url: 'Live Q&A'
+  },
+  {
+    day: 'Friday, July 22',
+    university: 'Noisy Real Hardware - Noise in Quantum Computers',
+    type: 'Zlatko Minev',
+    url: 'Lecture'
+  },
+  {
+    day: 'Friday, July 22',
+    university: 'Q&A Sessions',
+    type: '-',
+    url: 'Live Q&A'
+  }
+]
+
+// TODO: get full week 2 schedule from Events team
+const week2Schedule: eventSchedule[] = [
+  {
+    day: 'Monday, July 25',
+    university: 'Broad Overview of Quantum Chemistry Simulation and Why it is a Challenge',
+    type: 'Panos Barkoutsos',
+    url: 'Lecture'
+  },
+  {
+    day: 'Monday, July 25',
+    university: 'Q&A Sessions',
+    type: '-',
+    url: 'Live Q&A'
+  },
+  {
+    day: 'Tuesday, July 26',
+    university: 'Methods for Quantum Simulation',
+    type: 'Yukio Kawashima',
+    url: 'Lecture'
+  },
+  {
+    day: 'Tuesday, July 26',
+    university: 'Q&A Sessions',
+    type: '-',
+    url: 'Live Q&A'
+  },
+  {
+    day: 'Wednesday, July 27',
+    university: 'Quantum Static Simulation',
+    type: 'Ieva Liepuoniute',
+    url: 'Lecture'
+  },
+  {
+    day: 'Wednesday, July 27',
+    university: 'Q&A Sessions',
+    type: '-',
+    url: 'Live Q&A'
+  },
+  {
+    day: 'Thursday, July 28',
+    university: 'Quantum Dynamics',
+    type: 'Alexander Miessen',
+    url: 'Lecture'
+  },
+  {
+    day: 'Thursday, July 28',
+    university: 'Q&A Sessions',
+    type: '-',
+    url: 'Live Q&A'
+  },
+  {
+    day: 'Friday, July 29',
+    university: 'What is Quantum Simulation and Where is the Field Going?',
+    type: '-',
+    url: 'Lecture'
+  },
+  {
+    day: 'Friday, July 29',
+    university: 'Q&A Sessions',
+    type: '-',
+    url: 'Live Q&A'
+  },
+  {
+    day: 'Friday, July 29',
+    university: 'Qiskit Global Summer School 2022 Commencement & Wrap Celebration',
+    type: '-',
+    url: 'Commencement'
+  }
+]
+
+const scheduleToTableData = (slot: eventSchedule) => ([
+  {
+    styles: 'min-width: 10rem; display: inline-block; font-weight: bold;',
```

Comment:
this is interesting...is this the best way to apply styles? not familiar with this code base, but just expressing concern, as I'm sure we've all experienced trying to track down seemingly random styles that are abstracted from markup or follow a non-standard approach 
(☝️ comment not blocking)

Reply:
This is copied from an older approach that this code base had implemented, even going a couple of summers back, and it exists in a couple of other places in the code 🤔 

We can open code quality issue? but forgive me for thinking that we let it be for now.

Reply:
Yeah, seems reasonable to let it be...if this is a pattern we're using in other places in the code, doesn't seem worth trying to solution now. Thanks for the explanation @techtolentino !